### PR TITLE
Create viniciusredes.md

### PR DESCRIPTION
**Pull request by viniciusredes**
user:viniciusredes
time:17:06
quote:beber todo café que conseguir e pegar o máximo de brindes

### CODE REVIEW

**Review by github-education — CHANGES_REQUESTED** [review]
**Oie viniciusredes,**
**Bem vindo a Campus Party!**


### I have a few items I need you to take care of before I can merge this PR:

* *Por favor reveja seu arquivo markdown, possui alguns erros:* 
	*The attribute time is required in `_messages/viniciusredes.md`*
*The attribute time need to be in the format: YYYY-MM-DD HH:MM:SS. For example: 2022-06-15 17:01:10*
*You can't a time in the future.*
*The attribute `user` is required in `_messages/viniciusredes.md`*
*The attribute `time` is required in `_messages/viniciusredes.md`*
*The attribute `quote` is required in `_messages/viniciusredes.md`*

Feel free to re-request a review from me and I'll come back and take a look!

File: _messages/viniciusredes.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+user:viniciusredes
+time:2022-06-17 17:02
```
Comment (euaaron) [inline _messages/viniciusredes.md:2]:
```suggestion
time: 2022-06-17 17:02:00
```

File: _messages/viniciusredes.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+user:viniciusredes
```
Comment (euaaron) [inline _messages/viniciusredes.md:1]:
```suggestion
user: viniciusredes
```

Comment (euaaron) [reply _messages/viniciusredes.md:1]: Add a space

File: _messages/viniciusredes.md (hunk 1 above)
Comment (euaaron) [reply _messages/viniciusredes.md:2]: Adicionar espaço e os segundos

File: _messages/viniciusredes.md (hunk 2 above)
Comment (euaaron) [inline _messages/viniciusredes.md]:
```suggestion
---
user: viniciusredes
```

Comment (euaaron) [reply _messages/viniciusredes.md]: Adicionar hífens e espaço

File: _messages/viniciusredes.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+time:2022-06-17 17:02
+quote: Tomar quanto cafe conseguir e pegar o maximo de brindes
```
[the remark this answers is not in the document]
Comment (euaaron) [reply _messages/viniciusredes.md:3]: Adicionar hífens

File: _messages/viniciusredes.md [hunk 4]
```diff
@@ -0,0 +1,3 @@
+user: viniciusredes
```
Comment (euaaron) [inline _messages/viniciusredes.md:1]:
```suggestion
---
user: viniciusredes
```

Comment (euaaron) [reply _messages/viniciusredes.md:2]: Adicionar hífens antes e depois do conteúdo

File: _messages/viniciusredes.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
...
+time: 2022-06-17 17:02:00
+quote: Tomar quanto cafe conseguir e pegar o maximo de brindes
```
Comment (euaaron) [inline _messages/viniciusredes.md:4]:
```suggestion
quote: Tomar quanto cafe conseguir e pegar o maximo de brindes
---
```

Comment (euaaron) [reply _messages/viniciusredes.md:4]: Adicionar hífens na última linha

**Pull request merged**